explorer/client/src/pages/object-result/views/TokenView.tsx

```diff
@@ -0,0 +1,162 @@
+                                            isCopyButton={false}
+                                        />
+                                    </td>
+                                </tr>
+                            )}
+
+                            <tr>
+                                <td>Version</td>
+                                <td>{viewedData.version}</td>
+                            </tr>
+
+                            <tr>
+                                <td>Owner</td>
+                                <td id="owner">
+                                    <Longtext
+                                        text={
+                                            typeof viewedData.owner === 'string'
+                                                ? viewedData.owner
+                                                : typeof viewedData.owner
+                                        }
+                                        category="unknown"
+                                        isLink={
+                                            viewedData.owner !== 'Immutable' &&
+                                            viewedData.owner !== 'Shared'
+                                        }
+                                        isCopyButton={false}
+                                    />
+                                </td>
+                            </tr>
+                            {viewedData.contract_id && (
+                                <tr>
+                                    <td>Contract ID</td>
+                                    <td>
+                                        <Longtext
+                                            text={viewedData.contract_id.bytes}
+                                            category="objects"
+                                            isLink={true}
+                                        />
+                                    </td>
+                                </tr>
+                            )}
+                        </tbody>
+                    </table>
+                </div>
+                {viewedData.url !== '' && (
+                    <div className={styles.displaycontainer}>
+                        <div className={styles.display}>
+                            <DisplayBox display={viewedData.url} />
+                        </div>
+                        <div className={styles.metadata}>
+                            {name && <h2 className={styles.header}>{name}</h2>}
+                        </div>
+                    </div>
+                )}
+            </div>
+            {properties.length > 0 && (
+                <>
+                    <h2 className={styles.header}>Properties</h2>
+                    <table className={styles.properties}>
+                        <tbody>
+                            {properties.map(([key, value], index) => (
+                                <tr key={`property-${index}`}>
```

Comment:
Is the `key` in here unique? If so we should use that as a key. Otherwise, you can use the index directly without a string prefix.